supporting-blog-content/re-ranking-elasticsearch-hosted/re-ranking-elasticsearch-hosted.ipynb

```diff
@@ -0,0 +1,819 @@
+    "- The model has been deployed\n",
+    "\n",
+    "You should see JSON output with information about the semantic endpoint"
+   ],
+   "metadata": {
+    "id": "X8rQXMrHhMkS"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "check_endpoint = es.inference.get(\n",
+    "    inference_id=inference_id,\n",
+    ")\n",
+    "\n",
+    "check_endpoint.body"
+   ],
+   "metadata": {
+    "id": "n3Yk7rgYhP-N",
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "outputId": "d9e68225-5796-411e-964a-6db3be5541aa"
+   },
+   "execution_count": null,
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "{'endpoints': [{'inference_id': 'semantic-reranking',\n",
+       "   'task_type': 'rerank',\n",
+       "   'service': 'elasticsearch',\n",
+       "   'service_settings': {'num_allocations': 1,\n",
+       "    'num_threads': 1,\n",
+       "    'model_id': 'cross-encoder__ms-marco-minilm-l-6-v2'},\n",
+       "   'task_settings': {'return_documents': True}}]}"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 7
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Create the index mapping\n",
+    "\n",
+    "We are going to index the `title` and `abstract` from the dataset.  "
+   ],
+   "metadata": {
+    "id": "4vqimyNWAhWb"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "index_name = \"arxiv-papers\"\n",
+    "\n",
+    "index_mapping = {\n",
+    "    \"mappings\": {\n",
+    "        \"properties\": {\"title\": {\"type\": \"text\"}, \"abstract\": {\"type\": \"text\"}}\n",
+    "    }\n",
+    "}\n",
+    "\n",
+    "\n",
+    "try:\n",
+    "    es.indices.create(index=index_name, body=index_mapping)\n",
+    "    print(f\"Index '{index_name}' created successfully.\")\n",
+    "except exceptions.RequestError as e:\n",
+    "    if e.error == \"resource_already_exists_exception\":\n",
+    "        print(f\"Index '{index_name}' already exists.\")\n",
+    "    else:\n",
+    "        print(f\"Error creating index '{index_name}': {e}\")"
+   ],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "outputId": "4bc9ba0b-9be3-410a-d1d6-f1da04bbfec7",
+    "id": "DPADF_7ytTmR"
+   },
+   "execution_count": null,
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Index 'arxiv-papers' created successfully.\n"
+     ]
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Ready the dataset\n",
+    "We are going to use the [CShorten/ML-ArXiv-Papers](https://huggingface.co/datasets/CShorten/ML-ArXiv-Papers) dataset."
+   ],
+   "metadata": {
+    "id": "FqQmaT5P-Nhx"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Download Dataset\n",
+    "**Note** You may get a warking *The secret `HF_TOKEN` does not exist in your Colab secrets*.\n",
```

Comment:
Should this not be "warning" rather than "warking"? I raised this before but the comment was resolved with no correction or comment.

Let me know!

Reply:
Haha yes. Good catch, I missed it earlier. 
I pushed a correction